.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: objective-c
+osx_image: xcode10.1
+
+before_install:
+  - 'echo ''gem: --no-ri --no-rdoc'' > ~/.gemrc'
```

Comment:
Do we always want to override the ~/.gemrc ?

Reply:
I‘m just not sure whoelse mau write to gemrc. Let‘s keep it in mind and leave it like this

Reply:
Since this is only executed on the travis CI machine I guess we are pretty safe here. But yes, let's keep this in mind.